### PR TITLE
Implement client notification about search state changes over websocket

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

**Review by kirillta — APPROVED** [review]

**Review by mingazhev — COMMENTED** [review]
We'll discuss this additionally when I'll have more information

File: HappyTravel.Edo.NotificationCenter/Services/Notification/NotificationService.cs [hunk 1]
```diff
@@ -40,7 +41,7 @@ public async Task Add(Models.Notification notification)
                         => SendEmail(emailSettings),
                     
                     ProtocolTypes.WebSocket when settings is WebSocketSettings webSocketSettings 
-                        => _signalRSender.FireNotificationAddedEvent(notification.UserId, entry.Entity.Id, notification.Message),
+                        => SendWebSocket(notification.UserId, entry.Entity.Id, notification.Message),
```
Comment (mingazhev) [inline HappyTravel.Edo.NotificationCenter/Services/Notification/NotificationService.cs:44]: This is not strictly true. SignalR is not only websocket.
Maybe SendRealTimeNotification or something similar?

**Review by mingazhev — APPROVED** [review]
With a comment

**Comment by DFSko** [comment]
implemented in another branch

**Pull request closed without merging**